docs/core.html

```diff
@@ -9,17 +9,17 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
   <link rel="shortcut icon" href="favicon.ico">
   <title>reagent-query.reagent-query - Test helpers for reagent code</title>
-  <script src="js/highlight.min.js"></script>
```

Comment:
This file is autogenerated.
I'm using lucid.publish to generate the docs, and the JS and CSS files come from there.
I know this is not the best way to do things, but the way to autogenerate this file is a command written as a comment at the end of the project.clj file.
When running the command:
```bash
echo "(use 'lucid.publish) (copy-assets) (publish-all)" | lein repl
```
new `js` and `css` directories will be created, and the HTML is regenerated to point to them.